docs/docs/sign-in-out.md

```diff
@@ -2,8 +2,21 @@
 title: Sign In and Out
 layout: docs
 ---
+## Signing in and out
+To sign in and out of Tableau Server, simply call `Auth.sign_in` as follows:
+```py
+import tableauserverclient as TSC
+
+tableau_auth = TSC.TableauAuth('USERNAME', 'PASSWORD')
+server = TSC.Server('http://SERVER_URL')
+
+with server.auth.sign_in(tableau_auth):
+    # Do awesome things here!   
+
+# The Auth context manager will automatically call 'Auth.sign_out' on exiting the with block
```

Comment:
@RussTheAerialist may disagree with me here, but I don't think we should lead with the context manager.
I would lead by teaching `sign_in` and `sign_out` and then mentioning the context manger for simple scripts.

`with` blocks are intended for short lived transactions, RAII-style 'get-and-then-close', or similarly simple operations. Not long chains of REST API calls.

As evidence on this front, I've written a dozen TSC scripts for testing various other features my teams are interacting with, or configuring a server, etc, and I've never used the context manager because it baiscally automatically forces me out of pep8 for all but trivial code ... but that's my soapbox :)